src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -736,6 +736,18 @@ public void whenShouldSendRespondentNotification_andCaseIsConsented_thenShouldNo
         verify(notificationService, never()).sendContestOrderNotApprovedEmailRespondent(any());
     }
 
+    @Test
+    public void givenConsentedCase_whenSendConsentOrderNotApproved_thenShouldTriggerConsentedRespondentEmail() {
+        when(featureToggleService.isRespondentSolicitorEmailNotificationEnabled()).thenReturn(true);
```

Comment:
you'll need tests for the other logic flows here

Reply:
The other flow is tested in `givenContestedCase_whenShouldSendRespondentNotification_thenShouldTriggerRespondentEmail` - added a `verify` there to test the method is not getting called in contested case.